extras/rya.console/src/test/java/mvm/rya/shell/RyaAdminCommandsTest.java

```diff
@@ -123,6 +124,9 @@ public void deletePCJ() throws InstanceDoesNotExistException, RyaClientException
 
     @Test
     public void getInstanceDetails() throws InstanceDoesNotExistException, RyaClientException {
+        // This test is failed if the default timezone was not EST, so now it's fixed at EST.
+        // If you get assert mismatch of EST!=EDT, try the deprecated getTimeZone("EST") instead.
+        TimeZone.setDefault(TimeZone.getTimeZone("America/New_York"));
```

Comment:
Yes completely agree.  Java 8 or JodaTime.  They are mostly the same.  I would only use  java.util.Date if I had to.